CHANGELOG.md

```
@@ -1,3 +1,13 @@
+# [2.0.0](https://github.com/angelxmoreno/sneakerdb-client/compare/1.2.0...2.0.0) (2025-12-17)
+
+### Features
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)